var/spack/repos/builtin/packages/gearshifft/package.py

```diff
@@ -39,6 +40,11 @@ class Gearshifft(CMakePackage):
     # depends_on C++14 compiler, e.g. GCC 5.0+
     depends_on('cmake@2.8.0:', type='build')
     depends_on('boost@1.59.0:')
+
+    # TODO: replace this with an explicit list of components of Boost,
+    # for instance depends_on('boost +filesystem')
+    # See https://github.com/spack/spack/pull/22303 for reference
+    depends_on(Boost.with_default_variants)
```

Comment:
cc @zyzzyxdonta what kind of boost components does `gearshifft` use? This is still up-to-date?
https://github.com/mpicbg-scicomp/gearshifft/blob/b63be785d7cfd473bd5e14f70a124572962f24c6/CMakeLists.txt#L75-L77
https://github.com/mpicbg-scicomp/gearshifft/blob/b63be785d7cfd473bd5e14f70a124572962f24c6/CMakeLists.txt#L171-L174

Reply:
Boost::thread was added as an optional dependency after version 0.4.0 which is currently built by Spack. Version 0.4.0 links against
```
libboost_system
libboost_unit_test_framework
libboost_program_options
```
and includes
```
<boost/algorithm/string.hpp>
<boost/algorithm/string/predicate.hpp>
<boost/align/aligned_allocator.hpp>
<boost/asio.hpp>
<boost/bind.hpp>
<boost/config.hpp>
<boost/container/vector.hpp>
<boost/core/noncopyable.hpp>
<boost/mpl/for_each.hpp>
<boost/mpl/list.hpp>
<boost/noncopyable.hpp>
<boost/program_options.hpp>
<boost/range/counting_range.hpp>
<boost/test/unit_test.hpp>
<boost/test/unit_test_suite.hpp>
```

Reply:
thx!